assets/texts/zh-CN/amoScreenshots.csv

```diff
@@ -0,0 +1,8 @@
+qrBig.png; "QR codes are resizable!";
```

Comment:
These descriptions do also appear on AMO, so it would make quite sense to translate them, at least. :smile: